frontend/containers/dashboard/users/table/component.tsx

```diff
@@ -23,13 +23,13 @@ export const UsersTable: FC<UsersTableProps> = ({ isOwner }) => {
   const queryParams = useQueryParams();
 
   const {
-    data: { data: users } = { data: [] },
+    data: { data: users },
```

Comment:
```suggestion
    data: { data: users } = { data: [] },
```

Reply:
This removal breaks the page for me, especially when doing a refresh.
<img width="1680" alt="users" src="https://user-images.githubusercontent.com/6273795/180436463-7da075ff-c7c9-4392-8b11-3be64f1e03a8.png">

Reply:
I changed it for the exported {users} const